src/core/symbology/qgsmarkersymbollayer.cpp

```diff
@@ -3776,11 +3804,18 @@ void QgsFontMarkerSymbolLayer::renderPoint( QPointF point, QgsSymbolRenderContex
   const QString charToRender = characterToRender( context, chrOffset, chrWidth );
 
   const double sizeToRender = calculateSize( context );
+  double sizeToCalculateOffsets = sizeToRender;
+
+  if ( mVerticalAnchorMode == VerticalAnchorMode::Bounds )
+  {
+    // when we use the bounds, the font metrics used are already scaled. To be able to deal with this, they should not be scaled, so we scale them back beforehand.
+    sizeToCalculateOffsets = context.renderContext().convertFromPainterUnits( mFontMetrics->boundingRect( mString.at( 0 ) ).height(), mSizeUnit );
```

Comment:
This don't work with "Meter at Scale". It would work with converting, converting back and converting again. Don't know yet why...

```c++
    sizeToCalculateOffsets = context.renderContext().convertFromPainterUnits( mFontMetrics->boundingRect( mString.at( 0 ) ).height(), mSizeUnit );
    double newheight = context.renderContext().convertToPainterUnits(  sizeToCalculateOffsets, mSizeUnit);
    sizeToCalculateOffsets = context.renderContext().convertFromPainterUnits( newheight, mSizeUnit );
```